mailbox/store/src/main/java/org/apache/james/mailbox/store/probe/MailboxProbe.java

```diff
@@ -54,4 +54,6 @@ ComposedMessageId appendMessage(String username, MailboxPath mailboxPath, InputS
     void reIndexAll() throws Exception;
 
     Collection<String> listSubscriptions(String user) throws Exception;
+
+    void setMailboxACL(String namespace, String user, String mailboxName, String targetUser, String rights) throws MailboxException;
```

Comment:
I think we can use some types here

Reply:
We could discuss this.

The general practice seems to keep raw types on probes. The rationnal is serialization other JMX when using them for the CLI.

Should we change this practice?

Reply:
My opinion is that transport is transport (JMX is transport) and is basically managing serialization constraints. Then it calls an API with types.
Here, we are mixing both concerns and we end up with a bad API to use when not using JMX.
WDYT ?

Reply:
Sure. Agree.